provider/pkg/gen/dotnet-templates/helm/ChartBase.cs

```diff
@@ -309,11 +312,11 @@ private static string ExecuteCommand(string command, string[] flags, IDictionary
         /// <summary>
         /// Convert an argument array to an argument string for using with Process.StartInfo.Arguments.
         /// </summary>
-        private static string EscapeArguments(params string[] args)
-            => string.Join(" ", args.Select(EscapeArguments));
+        private static string EscapeArguments(IEnumerable<string> args)
+            => string.Join(" ", args.Select(a => EscapeArguments(a)));
```

Comment:
This is a subtle thing in C#. Specifying `args.Select(EscapeArguments)` will cause a new delegate to be allocated each time, but specifying it like `args.Select(a => EscapeArguments(a))` the compiler will cache the delegate.